tests/ci/travis-auth-server-test.sh

```diff
@@ -2,7 +2,7 @@
 
 # Install and start the auth server
 git clone https://github.com/mozilla/fxa-auth-server.git
-cd fxa-auth-server && npm i
+cd fxa-auth-server && git checkout verify-token-full && npm i
```

Comment:
This change should be reverted for the final merge. I want to ensure the tests pass with the auth server these changes go with.